snuba/subscriptions/executor_consumer.py

```diff
@@ -106,6 +107,20 @@ def get_topics_for_entity(
         strict_offset_reset=strict_offset_reset,
     )
 
+    try:
+        partition_count = get_partition_count(scheduled_topic_spec.topic)
+    except Exception:
+        logger.error("partition count unavailable..", exc_info=True)
+        partition_count = 0
```

Comment:
Is partition_count 0 a good default to fall back on? Or should we have some other default? 

In failure scenarios where a consumer is not able to find out the partition count, is the expected behavior going to be that the consumer will not not run at all?

Reply:
@nikhars I think for the purpose of this PR this is probably fine since we aren't using the `partition_count` yet, we are just logging it. But I think that if we can't get the partition_count that would be pretty problematic in that we couldn't accurately calculate the max_concurrent_queries.